kafka-rest/src/main/java/io/confluent/kafkarest/entities/ConsumerInstanceConfig.java

```diff
@@ -88,6 +116,68 @@ public String getName() {
     return name;
   }
 
+  @JsonProperty
+  public Integer getResponseMinBytes() {
+    return this.responseMinBytes;
+  }
+
+  @JsonProperty
+  public void setResponseMinBytes(String responseMinBytes) throws RestConstraintViolationException {
+    if (responseMinBytes == null) {
+      this.responseMinBytes = null;
+      return;
+    }
+
+    try {
+      this.responseMinBytes = Integer.parseInt(responseMinBytes);
+      KafkaRestConfig.PROXY_CONSUMER_RESPONSE_MIN_BYTES_VALIDATOR.ensureValid(
+              KafkaRestConfig.PROXY_FETCH_MIN_BYTES_CONFIG,
+              this.responseMinBytes
+      );
+    } catch (io.confluent.common.config.ConfigException e) {
+      throw new RestConstraintViolationException(
+              e.getMessage(), Response.Status.BAD_REQUEST.getStatusCode()
+      );
+    } catch (NumberFormatException e) {
+      throw new RestConstraintViolationException(
+              String.format("Invalid value for configuration %s: must be an integer",
+                      responseMinBytes,
+                      KafkaRestConfig.PROXY_FETCH_MIN_BYTES_CONFIG),
+              Response.Status.BAD_REQUEST.getStatusCode());
+    }
+  }
+
+  @JsonProperty
+  public Integer getRequestWaitMs() {
+    return this.requestWaitMs;
+  }
+
+  @JsonProperty
+  public void setRequestWaitMs(String requestWaitMs) throws RestConstraintViolationException {
+    if (requestWaitMs == null) {
+      this.requestWaitMs = null;
+      return;
+    }
+
+    try {
+      this.requestWaitMs = Integer.parseInt(requestWaitMs);
+      KafkaRestConfig.PROXY_FETCH_MAX_WAIT_MS_VALIDATOR.ensureValid(
+              KafkaRestConfig.PROXY_FETCH_MAX_WAIT_MS_CONFIG,
+              this.requestWaitMs
+      );
+    } catch (io.confluent.common.config.ConfigException e) {
+      throw new RestConstraintViolationException(
```

Comment:
Is it helpful to have an error code like we have for consumer config exceptions?